### ISSUE TITLE
Add support for exemptions in SCI/ATC

### ISSUE THREAD

**Issue by GiantCrocodile** [abapGit/abapGit#4109]
Hello,

it would be great to have support to view and to request exemptions for SCI/ATC within abapGit.

Related to #3935

Best Regards,

**Comment by flaiker** [comment]
I am not sure if it makes sense to reimplement all features of the ATC result browser into abapGit. The current integration is already kind of a trade of between a nice usability feature from a workflow perspective and something not belonging at all in a git client.

**Comment by GiantCrocodile** [comment]
I agree regarding your concerns about this. Reimplementing all these features would be too much work for too less benefit. Offering this and other related features as a Git client is nice as it make abapGit a great start entry to implement a CI/CD process in your development process. 

A better idea might be to not implement all the ATC stuff but offer the user a shortcut way like a link or something to jump directly to the ATC result browser to use the existing SAP standard features. Alternatively we could embed the ATC result browser information inside abapGit. I dunno if one of the two ways is possible or if it would be lot of work. What do you think @flaiker?

**Comment by flaiker** [comment]
I think linking to other stuff generally is a good approach. Though incompatible with the web based abapGit endeavors. I don't know how the current integration works. If a named run is executed you could probably open the result list in the standard transaction without rerunning.

I assume you are using exemptions on _any_ ATC run and not just on the central run?

**Comment by GiantCrocodile** [comment]
Yes, we have exemptions enabled on local and remote ATC checks but we do use only central/remote ATC checks primarily.

**Comment by mbtools** [comment]
Here's how to get the exemptions, in case you want to implement this:

`SAPLSATC_CI_EXEMPTION_BROWSER  / LSATC_CI_EXEMPTION_BROWSERP01
METHOD          / PAI (SELECTION_SCREEN)`

```abap
        query_api = cl_satc_ci_xmpt_api_factory=>get_exemption_query_api( ).
        query_api->get_exemptions(
          exporting    i_package_filter            = package_filter
          importing   e_exemptions                = fg_exemptions ).
```

You will have to check if this exists in 702.
